Documentation/compatibility/wpf-databinding-should-use-key-not-indexer.md

```diff
@@ -0,0 +1,40 @@
+## Data Binding improvement for KeyedCollection
+
+### Scope
+Major
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Fixed [Binding's](xref:System.Windows.Data.Binding) incorrect use of IList indexer when the source object declares a custom indexer with the same signature (e.g. KeyedCollection<int,TItem>).
+
+- [ ] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+In order for the application to benefit from these changes, it must run on the .NET Framework 4.7.2 or later and it must opt-in to enable by setting the following [AppContext Switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the `<runtime>` section of the app config file and setting it to `false`, as the following example shows.
```

Comment:
Is 4.7.2 correct. If so, this should be a separate PR that merges to the master branch, not the net48 branch.

```suggestion
In order for the application to benefit from this change, it must run on the .NET Framework 4.7.2 or later, and it must opt in to the change by adding the following [AppContext switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the `<runtime>` section of the app config file and setting it to `false`:
```